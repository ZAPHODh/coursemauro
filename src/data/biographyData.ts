export const biographyData = [
    `Comecei minha carreira como cabeleireiro há 19 anos, em um cômodo minúsculo de uma barbearia de bairro, focando inicialmente em cortes masculinos. Desde o início, eu sabia que precisaria batalhar muito para alcançar meus objetivos. Foi nessa época que comecei a estudar técnicas de iluminação, usando amigos e familiares como modelos para testar, aprender e obter bons resultados.`,
    `Com muita prática e estudo, fui me aperfeiçoando cada vez mais. Eventualmente, outros profissionais começaram a me perguntar: "Como você consegue esses resultados?" Depois de ouvir essa pergunta inúmeras vezes e ajudar muitos colegas, percebi que poderia facilitar o caminho de outros profissionais.`,
    `Assim, comecei a dar aulas e cursos, compartilhando minhas experiências e técnicas. Meu objetivo é ajudar outros cabeleireiros a evitarem as dificuldades que enfrentei no início da minha carreira, capacitando-os a alcançarem o sucesso com mais facilidade e rapidez.`,
]
