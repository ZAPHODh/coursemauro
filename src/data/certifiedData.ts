import certifiedImg from '@/../public/images/certified.webp'
export const certifiedData = {
    heading: 'Certificado',
    img: certifiedImg,
    description:
        'Ao finalizar o curso, voce imediatamente terá acesso a um certificado de conclusão.',
}
