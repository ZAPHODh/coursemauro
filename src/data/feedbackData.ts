import feedback1 from '@/../public/images/feedback/1.jpeg'
import feedback2 from '@/../public/images/feedback/2.jpeg'
import feedback3 from '@/../public/images/feedback/3.jpeg'
import feedback4 from '@/../public/images/feedback/4.jpeg'
import feedback5 from '@/../public/images/feedback/5.jpeg'
import feedback6 from '@/../public/images/feedback/6.jpeg'
import feedback7 from '@/../public/images/feedback/7.jpeg'
import feedback8 from '@/../public/images/feedback/8.jpeg'
import feedback9 from '@/../public/images/feedback/9.jpeg'
import feedback10 from '@/../public/images/feedback/10.jpeg'
import feedback11 from '@/../public/images/feedback/11.jpeg'
import feedback12 from '@/../public/images/feedback/12.jpeg'
import feedback13 from '@/../public/images/feedback/13.jpeg'
import feedback14 from '@/../public/images/feedback/14.jpeg'
import feedback15 from '@/../public/images/feedback/15.jpeg'
import feedback16 from '@/../public/images/feedback/16.jpeg'
import feedback17 from '@/../public/images/feedback/17.jpeg'
import feedback18 from '@/../public/images/feedback/18.jpeg'
import feedback19 from '@/../public/images/feedback/19.jpeg'
import feedback20 from '@/../public/images/feedback/20.jpeg'
import feedback21 from '@/../public/images/feedback/21.jpeg'
import feedback22 from '@/../public/images/feedback/22.jpeg'
import feedback23 from '@/../public/images/feedback/23.jpeg'
import feedback24 from '@/../public/images/feedback/24.jpeg'
import feedback25 from '@/../public/images/feedback/25.jpeg'

export const feedbackData = [
    feedback1,
    feedback2,
    feedback3,
    feedback4,
    feedback5,
    feedback6,
    feedback7,
    feedback8,
    feedback9,
    feedback10,
    feedback11,
    feedback12,
    feedback13,
    feedback14,
    feedback15,
    feedback16,
    feedback17,
    feedback18,
    feedback19,
    feedback20,
    feedback21,
    feedback22,
    feedback23,
    feedback24,
    feedback25,
]
