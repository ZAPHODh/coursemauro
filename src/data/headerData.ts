export const headerData = {
    course: 'Iluminados sem segredos',
    description:
        'Iluminados sem segredos é um curso para quem quer aprender uma técnica que prioriza a economia de papéis e produtos, obtendo um resultado natural e elegante.',
}
