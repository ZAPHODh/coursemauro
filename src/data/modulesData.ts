export const modules = [
    { name: 'INTRODUÇÃO', number: 1 },
    { name: 'MONTAGEM', number: 2 },
    { name: 'ERIÇADO', number: 3 },
    { name: 'TÉCNICA', number: 4 },
    { name: 'sleep', number: 6 },
]
