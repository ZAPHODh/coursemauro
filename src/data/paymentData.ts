export const paymentData = {
    value: 165,
    benefits: [
        'beneficio ',
        'beneficio ',
        'beneficio ',
        'beneficio ',
        'beneficio ',
        'beneficio ',
    ],
}
