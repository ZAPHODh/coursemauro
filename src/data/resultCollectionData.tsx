import model1 from '../../public/images/models/model1.jpeg'
import model2 from '../../public/images/models/model2.jpeg'
import model3 from '../../public/images/models/model3.jpeg'
import model4 from '../../public/images/models/model4.jpeg'
export const resultCollectionData = {
    title: 'Resultados que voce irá alcançar',
    images: [
        { img: model1, alt: 'mostrando o cabelo da modelo' },
        { img: model2, alt: 'mostrando o cabelo da modelo' },
        { img: model3, alt: 'mostrando o cabelo da modelo' },
        { img: model4, alt: 'mostrando o cabelo da modelo' },
    ],
}
