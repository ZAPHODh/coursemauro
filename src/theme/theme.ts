import { DefaultTheme } from 'styled-components'

export const theme: DefaultTheme = {
    colors: {
        white: '#ffffff ',
        darker: '#0B0B0D',
        dark: '#181619',
        normalDark: '#181B22',
        brightDark: '#272A31',
        gold: '#A76B09',
        brightGold: '#DDA74F',
    },
}
