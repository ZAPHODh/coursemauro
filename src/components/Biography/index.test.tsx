import { render } from '@testing-library/react'
import { Biography } from '.'
import { ThemeClient } from '../../theme/ThemeClient'
import { biographyData } from '@/data/biographyData'

describe('Biography', () => {
    it('should render the Biography', () => {
        const { getByText } = render(
            <ThemeClient>
                <Biography biography={biographyData} />
            </ThemeClient>
        )

        const component = getByText(biographyData[0])

        expect(component).toBeInTheDocument()
    })
    it('should render match snapshot', () => {
        const { getByText } = render(
            <ThemeClient>
                <Biography biography={biographyData} />
            </ThemeClient>
        )

        const component = getByText(biographyData[0])

        expect(component).toMatchSnapshot()
    })
})
