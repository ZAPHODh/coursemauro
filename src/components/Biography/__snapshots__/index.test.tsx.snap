// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Biography should render match snapshot 1`] = `
<p
  class="sc-69380e9b-0 dUhRVp"
>
  Comecei minha carreira como cabeleireiro há 19 anos, em um cômodo minúsculo de uma barbearia de bairro, focando inicialmente em cortes masculinos. Desde o início, eu sabia que precisaria batalhar muito para alcançar meus objetivos. Foi nessa época que comecei a estudar técnicas de iluminação, usando amigos e familiares como modelos para testar, aprender e obter bons resultados.
</p>
`;
